### PR TITLE
create Home Screen, Trending movie

### PR DESCRIPTION
**Pull request by NVManh246**
https://edu-redmine.framgia.vn/issues/1251

### CODE REVIEW

**Comment by NVManh246** [comment]
ready

File: app/src/main/res/layout/fragment_home.xml [hunk 1]
```diff
@@ -1,18 +1,126 @@
...
+                app:layout_constraintTop_toBottomOf="@+id/viewpager"
+                android:text="@string/title_toprated"
+                android:background="@drawable/action"
+                android:textColor="@android:color/white"
+                android:textStyle="bold"
+                android:textSize="@dimen/sp_20"
+                />
+
+            <android.support.constraint.Guideline
+                android:id="@+id/guideline2"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/fragment_home.xml]: Convention 3.1. rename to guide_line_two or guide_line_second

File: app/src/main/res/layout/fragment_home.xml [hunk 2]
```diff
@@ -1,18 +1,126 @@
...
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:orientation="vertical"
+                app:layout_constraintGuide_percent="0.5" />
+
+            <Button
+                android:id="@+id/imageView4"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/fragment_home.xml]: Convention 3.1`

File: app/src/main/res/layout/fragment_home.xml [hunk 3]
```diff
@@ -1,18 +1,126 @@
...
+        android:layout_height="match_parent">
+
+        <android.support.constraint.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@color/color_cod_gray"
+            >
+
+            <android.support.v4.view.ViewPager
+                android:id="@+id/viewpager"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/fragment_home.xml]: Convention 3.1. view_pager

File: app/src/main/res/layout/item_trending_movie.xml [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+            android:layout_marginLeft="@dimen/dp_5"
+            android:layout_marginStart="@dimen/dp_5"
+            android:layout_marginTop="@dimen/dp_10"
+            android:ellipsize="end"
+            android:maxLines="3"
+            android:text="@string/title_time"
+            android:textColor="@android:color/white"
+            android:textSize="@dimen/sp_12"
+            android:textStyle="bold"
+            app:layout_constraintStart_toStartOf="@+id/guideline"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/item_trending_movie.xml]: guide_line

**Pull request merged**